frontend/src/components/Page.js

```diff
@@ -2,7 +2,6 @@ import styled from "styled-components/macro";
 
 export default styled.div`
   position: fixed;
-  top: 0;
   left: 0;
```

Comment:
Now I am wondering why you don't set top 🤔

Reply:
Because Header and navigator is now in app.js. Page should be renamed to main now. Because it's just for the main part now.

Reply:
Ah, I see. I agree, main would be a better name. Not so sure about layouting with the fixed position, though. I would have used a wrapper component for the page, but if it works, why not ;)